### DISCUSSION TITLE
Is there a way to add the old button into new jsx format?

### DISCUSSION THREAD

**Discussion by pranavcpawar** [MetaMask/snaps#2608] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/b2e63f73-b0f6-47ae-99be-32fd0ae7ae8c)
Can I get this button in new JSX button?

**Comment by Montoya** [comment, marked as answer]
No, we have decided to change the button style for Snaps, but soon we will introduce a new footer component that can have the pill-style buttons in the footer of a dialog or homepage.
